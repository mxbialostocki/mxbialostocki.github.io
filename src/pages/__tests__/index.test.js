import React from "react"
import ShallowRenderer from "react-test-renderer/shallow"

import Index from "../index"

describe("Index Page", () => {
  const renderer = new ShallowRenderer()
  it("renders correctly", () => {
    const tree = renderer.render(
    <Index />
    )
    expect(tree).toMatchSnapshot()
  })
})
