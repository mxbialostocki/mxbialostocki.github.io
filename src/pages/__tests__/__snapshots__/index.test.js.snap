// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page renders correctly 1`] = `
<React.Fragment>
  <Layout>
    <SEO
      description=""
      lang="en"
      meta={Array []}
      title="Home"
    />
    <WithStyles(ForwardRef(Grid))
      container={true}
      direction="column"
      spacing={4}
    >
      <WithStyles(ForwardRef(Grid))
        item={true}
      >
        <WithStyles(ForwardRef(Typography))
          style={
            Object {
              "paddingBottom": "1rem",
            }
          }
          variant="h2"
        >
          hello, Creature
        </WithStyles(ForwardRef(Typography))>
        <WithStyles(ForwardRef(Typography))
          variant="h5"
        >
          My name is Bam.
        </WithStyles(ForwardRef(Typography))>
        <WithStyles(ForwardRef(Typography))
          className="makeStyles-primary-1"
          variant="h5"
        >
          I am the dev you're looking for.
        </WithStyles(ForwardRef(Typography))>
      </WithStyles(ForwardRef(Grid))>
    </WithStyles(ForwardRef(Grid))>
    <Pages />
  </Layout>
</React.Fragment>
`;
